vre/values-custom.yaml

```diff
@@ -0,0 +1,7 @@
+reana:
```

Comment:
Wishlist to add
hostbase or similar, that is templated for the jhub and reana host names
Make indigo IAM client secret available for jhub too
Indigo IAM authorise, token and callback URLs for reana and jhub
nfs storage classes, both the default for persistent, and the reana shared storage class
common (postgres)db secret
Rucio host (template the extraEnv and rucio.cfg file generation ?)

Reply:
Note that it is not that straightforward to remove all duplication from values. It is not possible to use template within values file itself.
It is possible to use yaml anchors but it's not for all cases (e.g. not when substring need to be templated).

Usually to avoid duplication like that, it is necessary to modify the chart itself, in this case the dependency charts.
I will see what can be done without being too invasive.

We could also template values ourselves, but it means adding a custom tool on top of helm and this will be less portable.